src/parameters.toml

```diff
@@ -275,6 +275,109 @@ value = 1
 type = "float"
 description = "Exponent used in β decay (unitless). Source: SLIM model (Laguë et al 2019)"
 
+# Snow 
+
+[snow_density]
+alias = "ρ_snow"
+value = 200
+type = "float"
+description = "Density of snow (kg/m^3)"
+
+[snow_albedo]
+alias = "α_snow"
+value = 0.8
+type = "float"
+description = "Albedo of snow (unitless)"
+
+[snow_emissivity]
+alias = "ϵ_snow"
+value = 0.97
+type = "float"
+description = "Emissivity of snow (unitless)"
+
+[holding_capacity_of_water_in_snow]
+alias = "θ_r"
+value = 0.08
+type = "float"
+description = "Volumetric holding capacity of water in snow (unitless)"
+
+[wet_snow_hydraulic_conductivity]
+alias = "Ksat"
+value = 1e-3
+type = "float"
+description = "Hydraulic conductivity of wet snow (m/s)"
+
+[snow_cover_fraction_crit_threshold]
+alias = "fS_c"
+value = 0.2
+type = "float"
+description = "Critical threshold for snow cover fraction (m)"
+
+# Two Stream/Beer Lambert Parameters
+
+[leaf_angle_distribution_function]
```

Comment:
I believe this is ultimately coming from maps too and is no longer a fixed constant for all vegetation.